fuzz/fuzz_targets/coprocessor/codec/mysql/decimal.rs

```diff
@@ -43,9 +43,6 @@ where
     let _ = lhs.is_zero();
     let _ = lhs.approximate_encoded_size();
 
-    let frac_inc = iter.next().unwrap();
-    lhs.clone().div(rhs.clone(), frac_inc);
```

Comment:
Why remove `div`?

Reply:
The function `div` is private and the `frac_inc` is limited